packages/components/src/components/form/AutocompleteMultiSelect/AutocompleteMultiSelect.stories.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { Meta } from '@storybook/react';
+import { autocompleteMultiSelectSizes } from './types';
+import { AutocompleteMultiSelect } from './AutocompleteMultiSelect';
+import { Table, Tbody, Td, Th, Thead, Tr } from '@chakra-ui/react';
+
+export default {
+  component: AutocompleteMultiSelect,
+  title: 'Components/Form/AutocompleteMultiSelect',
+  argTypes: {
+    size: {
+      options: autocompleteMultiSelectSizes,
+      control: { type: 'select' },
+    },
+  },
+} as Meta;
+
+const defaultItems = [
```

Comment:
Could we try using an array of objects instead? I'd like to ensure we still have that functionality working 🙇🏻

Reply:
done!